### PR TITLE
Add npm-scripts for individual tests

### PR DESCRIPTION
**Pull request by phixid**
Added some npm-scripts to easily run individual tests.

### CODE REVIEW

File: __tests__/07_lenses.test.js [hunk 1]
```diff
@@ -219,5 +219,9 @@ describe('Lenses', () => {
     test('It should allow us to set values at index x', () => {
       expect(first.set(42, mockValues)).toEqual([42, 1, 2]);
     });
+
+    test('It should be able to over a value at a given index', () => {
```
Comment (0xClpz) [inline __tests__/07_lenses.test.js]: Great !! Thank you very much ! Will merge after fixing my next comment !

File: package.json [hunk 2]
```diff
@@ -16,6 +16,14 @@
   },
   "scripts": {
     "test": "jest --watch",
+    "test:1": "jest __tests__/01*.test.js  --watch",
```
Comment (0xClpz) [inline package.json]: this can already be achieved by doing *yarn test 1*

File: __tests__/07_lenses.test.js (hunk 1 above)
Comment (phixid) [reply __tests__/07_lenses.test.js]: Thanks man! Great course by the way.

File: package.json (hunk 2 above)
Comment (phixid) [reply package.json]: Amazing, didn't know! Thanks!

**Comment by 0xClpz** [comment]
Thank you for your contribution !

**Pull request merged**